internal/driver/webhtml.go

```diff
@@ -370,6 +370,12 @@ table tr td {
     </div>
   </div>
 
+  <div id="download" class="menu-item">
+    <div class="menu-name">
+      <a href="./download">Download</a>
```

Comment:
Could we style this so that it doesn't look like a hyperlink in the UI? On the screenshot included in the PR description the new element stands out a lot, more than it should as it's not the most important action.

Reply:
I can help with styling this. Do you want the new element looks like other elements, just doesn't have the dropdown?

Reply:
Yes, that would work. Or make it an item in the "View" menu. It's not exactly a view, but I think it's close enough to be there.

Reply:
hi @aalexand I created the PR linked below. Could you please check?

Reply:
Merged #634. Thanks!